lessons/4_numbers/index.html

```diff
@@ -249,6 +255,105 @@ <h3>Add calculator v2</h3>
+                    </code>
+                </pre>
+                <script>
+                    (() => {
+                        const inputOneEl = document.getElementById('addOneInputv3');
+                        const updateBtnEl = document.getElementById('updateAnswerv3');
+                        const answerEl = document.getElementById('addAnswerv3');
+
+                        let sum = 0; // global scope
+
+                        function handleInputChange(event) {
+                            const num = parseFloat(event.target.value); // local scope
+                            sum = sum + num;
+                        }
+
+                        inputOneEl.addEventListener('change', handleInputChange);
+
+                        function updateAnswer() {
+                            answerEl.textContent = sum;
+                            console.log('sum:', sum);
+                            console.log('num:', num); // will error because there is no num in this scope
+                        }
+
+                        updateBtnEl.addEventListener('click', updateAnswer);
+                    })();
+                </script>
+            </section>
+            <section>
+                <h3>Animal List Example</h3>
+                <ul id="animalList">
+                    <li>dog</li>
+                </ul>
+                <div class="input_group">
+                    <div class="input-group__input-wrapper"><input id="animalInput" placeholder="animal" /></div>
+                    <button type="button" id="addAnimal">add animal</button>
+                </div>
+                <script>
+                    (() => {
+                        const animalListEl = document.getElementById('animalList');
+                        const animalInputEl = document.getElementById('animalInput');
+                        const animalAddBtn = document.getElementById('addAnimal');
+
+                        function addAnimal() {
+                            const animal = animalInputEl.value;
+                            const animalEl = document.createElement('li');
+                            animalEl.textContent = animal;
+                            animalListEl.appendChild(animalEl);
+                            animalInputEl.value = '';
+                        }
+
+                        animalAddBtn.addEventListener('click', addAnimal);
+                    })();
+                </script>
+            </section>
+            <section>
+                <h3>Animal List Example Code</h3>
+                <pre>
+                    <code>
+const animalListEl = document.getElementById('animalList');
+const animalInputEl = document.getElementById('animalInput');
+const animalAddBtn = document.getElementById('addAnimal');
+
+function addAnimal() {
+    const animal = animalInputEl.value;
+    const animalEl = document.createElement('li');
+    animalEl.textContent = animal;
+    animalListEl.appendChild(animalEl);
```

Comment:
crateElement and appendChild needs to be listed maybe on a separate slide since it's new methods to manipulate with the DOM. What do you think?

Reply:
Good idea added a slide to explain them.